CHANGELOG.md

```diff
@@ -9,7 +9,9 @@ All notable changes for each version of this project will be documented in this
     - **Breaking Change** - The `locale` and `pipeArgs` parameters are removed from the `operate` method exposed by the `IgxNumberSummaryOperand`, `IgxDateSummaryOperand`, `IgxCurrencySummaryOperand` and `IgxPercentSummaryOperand`. They are now set in the `igx-grid-summary-cell` template. To change the locale and format setting of the `igx-grid-summary-cell` the user can use the new `summaryFormatter` property of the `IgxColumnComponent`.
 - `IgxTabs`, `IgxBottomNav`
     - **Breaking Change** - `IgxTabs` and `IgxBottomNav` components were completely refactored in order to provide more flexible and descriptive way to define tab headers and contents. Please make sure to update via `ng update` in order to migrate the existing `igx-tabs` and `igx-bottom-nav` definitions to the new ones.
-  - **Breaking Change** - `IgxHierarchicalGrid` and `igxRowIsland` events are renamed as follows:
+- `IgxColumnComponent`
+    - **Breaking Change** - The `onColumnChange` output is renamed to `columnChange`.
```

Comment:
@dobromirts @kdinev 
Guys, `[on]ColumnChange` is a hidden event AFAIK; internally used by the H Grid's `WatchColumnChanges` to pass settings to child grids. As such it really doesn't warrant a breaking entry in the changelog or migration for that matter :)